src/main/java/duke/Main.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+
+
+public class Main extends Application {
+
+    private Duke duke = new Duke("data/duke.txt");
+
+    @Override
+    public void start(Stage stage) {
```

Comment:
Saw this in a few places, remember to add header comments for all public methods!